### ISSUE TITLE
error while training.

### ISSUE THREAD

**Issue by BlcaKHat** [plabadille/image_classifier#2]
![validation](https://user-images.githubusercontent.com/16970999/45361516-070ef780-b5f0-11e8-9c8c-147a82eac9db.png)

I got this error .

**Comment by plabadille** [comment]
Hello, 

it seem your validation_sample_count is undefined (e.g None in python). This mean you probably have a format issue with your hdf5 file. Have you generate your hdf5 file using my script `serialize_dataset_to_hdf5.py` ?

Note h5 weights file and a serialized dataset in hdf5 ARE NOT the same things

**Comment by BlcaKHat** [comment]
yeah, I generated using your script. and text file was similar to what you mentioned.

**Comment by plabadille** [comment]
Are you using the last version of the scripts on the repository ? I have
done fixes since the release

Otherwise :
- check you're using the rigth versions (
https://github.com/plabadille/image_classifier#packages-version-used) i
didn't try it with newer version of Tensorflow/Keras.
- provide me all the constants you defined at the begining of the scripts
- make sure your hdf5 file path is correct and the file is loaded

Le mar. 11 sept. 2018 à 19:27, vijay singh <notifications@github.com> a
écrit :

**Comment by plabadille** [comment]
I can see on your screenshot `/Documents/image_classifier/serialized_datasets/helmet`
Your serialized dataset filename should be : `helmet.hdf5` (it seem you named it `helmet`)
Then you should call the script like this : `python train_with_hdf5.py helmet`

**If this is not the issue :**
Are you using the last version of the scripts on the repository ? I have
done fixes since the release

**Otherwise :**
* check you're using the rigth versions (
https://github.com/plabadille/image_classifier#packages-version-used) i
didn't try it with newer version of Tensorflow/Keras.
* provide me all the constants you defined at the begining of the scripts
* make sure your hdf5 file path is correct and the file is loaded
